.gitignore

```diff
@@ -2,6 +2,9 @@
 .Rhistory
 .RData
 .Ruserdata
+*.DS_Store
 
 # Specific files/dirs to ignore for this pipeline
+_targets/*
 */tmp/*
+*/out*
```

Comment:
Whoops looks like you're missing a `/` here -- while I think this may work as is to ignore anything in a `out/` subdirectory (thought I'm not 100% sure that `*` can match a slash), I believe it could theoretically exclude files with `out` in their name within any `1_fetch` or `2_process` subdirectory, e.g., `2_process/out_data.csv`. I think to guarantee that only the contents of the `out` _directory_ within those subdirectories is ignored, this will need to be `*/out/*`